python/setup_project/wizard.py

```diff
@@ -113,50 +112,50 @@ def _on_help_requested(self):
         page = self.currentPage()
         page.help_requested()
 
-    def _is_store_valid(self, store_info):
-        """ returns True if the store should be valid.  False otherwise """
-        return store_info["exists_on_disk"] and store_info["defined_in_shotgun"]
+    def validate_config_uri(self, uri):
+        """Download and validate the supplied config URI.
 
-    def set_config_uri(self, uri):
```

Comment:
This is the old logic to handle deferring setting the config uri as mentioned above.